### ISSUE TITLE
why is the recursion here?

### ISSUE THREAD

**Issue by urmyfaith** [onevcat/DispatchMemoryLeakDemo#1]
why is the recursion [here](https://github.com/onevcat/DispatchMemoryLeakDemo/blob/master/DispatchMemoryLeakDemo/ViewController.swift#L25)?

**Comment by urmyfaith** [comment]
just to create enough queue?

**Comment by onevcat** [comment]
Yep. To produce the memory increasing fast enough for demonstrating purpose.
